ajax/libs/ef.js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "ef.js",
+  "description": "(maybe) An elegant HTML template engine & basic framework",
+  "filename": "ef.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/TheNeuronProject/ef.js.git"
+  },
+  "keywords": [
+    "template",
+    "framework",
+    "ef",
+    "ef.js"
+  ],
+  "author": "ClassicOldSong",
+  "license": "MIT",
+  "homepage": "https://ef.js.org",
+  "npmName": "ef.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
Let's modify to `"**/!(*.gz)"`
Because current pattern can get the `ef.min.gz` from v0.1.3-alpha.4
And this file is not useful to the front-end developer.
Thank you.